BUILD

```diff
@@ -28,3 +28,21 @@ gazelle(
 
 # This could be any file, used as an anchor point for the directory in tests
 exports_files(["README.md"])
+
+go_path(
+    name = "all_srcs",
+    deps = [
+        "//go/tools/builders:asm",
```

Comment:
Right now there is no good automated way to generate that list....
In theory adding all test and binary targets to the list ought to be sufficient for most cases, but there is nothing to stop you having multiple of these rules with differing subsets of the dependency list, and if you have libraries with no tests you expose for other repositories to consume they would not be covered by that either.
For now, lets leave it fully manual until we know more about how people want to use it.